internal/controller/config/crd/bases/adi.io.my.domain_agents.yaml

```diff
@@ -268,6 +272,8 @@ spec:
                           type: string
                         wwn:
                           type: string
+                      required:
+                      - id
```

Comment:
Making it required turns it into a pointer, make sure the agent is prepared for that